pom.xml

```diff
@@ -18,6 +18,10 @@
             <id>public.onebusaway.org</id>
             <url>http://nexus.onebusaway.org/content/groups/public/</url>
         </repository>
+        <repository>
```

Comment:
I don't see any downsides switching to provider like MySQL, rather it has performance improvements.
One link I pulled online http://www.jpab.org/Hibernate/MySQL/server/Hibernate/SQLite/embedded.html

Shall we use mysql instead sqlite? 
If we switch, for now what I can see are, some query changes.